bleak/backends/winrt/client.py

```diff
@@ -207,7 +207,13 @@ def _create_requester(self, bluetooth_address: int):
                 if self._address_type == "public"
                 else BluetoothAddressType.RANDOM
             )
-        return BluetoothLEDevice.from_bluetooth_address_async(*args)
+        requester = BluetoothLEDevice.from_bluetooth_address_async(*args)
+        if requester is None:
+            raise BleakDeviceNotFoundError(
+                BLEDevice(self.address),
```

Comment:
Sorry I didn't think about this before when I suggested using `BLEDevice`, but if we don't have any other information besides the address, then it doesn't make sense to create an incomplete `BLEDevice` object. So probably better to just stick with the "identifier" as you originally had it.

Reply:
No problem. I changed it back to the identifier version. 
Nevertheless storing the whole BLEDevice might be an interesting improvement for the future.